kolibri/plugins/management/assets/src/device_management/views/select-content-page/selected-resources-size.vue

```diff
@@ -96,21 +99,25 @@
 
 <style lang="stylus" scoped>
 
+  .selected-resources-size
+    // using table to separate element by alignment while keeping them on the same line
+    // avoids magic numbers, keeps text lined up.
```

Comment:
[pure grids](https://purecss.io/grids/) might also be a good solution for a case like this. I'd like to try to start using them a bit more

Reply:
I tried it at first, but they're still just floated divs, which meant the `inline` line wasn't consistent

Reply:
interesting. would love to see a demo at some point so I can understand the limitations of pure grides